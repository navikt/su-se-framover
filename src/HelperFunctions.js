// endrer på tekstinnholdet for oppsumerings-siden
export const jaNeiSpørsmål = state => {
    if (state) {
        return 'Ja';
    } else if (state === false) {
        return 'Nei';
    } else {
        return '';
    }
};

//Dato input fra state "2020-12-25" -> "25-12-2020"
export const reverseString = str => {
    if (str !== undefined) {
        const splitString = str.split('-');
        const reverseArray = splitString.reverse();
        const joinedString = reverseArray.join('-');
        return joinedString;
    }
};

//replaces white spaces and dots in an array with nothing
//only to be used for visual presentation for numbers.
//array should contain an object with a field named 'beløp'
export const replace = arr => {
    const newArray = [];
    if (arr.length > 0) {
        arr.forEach(obj => newArray.push(obj.beløp.replace(/\s/g, '').replace(/\./g, '')));
        return newArray;
    }
};

export const isStringBoolean = target => {
    return typeof target === 'string' && (target.toLowerCase() === 'true' || target.toLowerCase() === 'false');
};

export const stringToBoolean = value => {
    return value === 'true';
};

export const displayErrorMessageOnInputField = (feilmeldinger, skjemaElementLabel) => {
    let melding = "";
    feilmeldinger.forEach(obj =>{
        if(obj.skjemaelementId === skjemaElementLabel){
            melding = obj.feilmelding;
        }
    });
    return melding;
};