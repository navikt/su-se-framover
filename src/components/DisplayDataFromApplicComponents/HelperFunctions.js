// endrer på tekstinnholdet for oppsumerings-siden
export const jaNeiSpørsmål = state => {
    if (state) {
        return 'Ja';
    } else if (state === false) {
        return 'Nei';
    } else {
        return '';
    }
};

//Dato input fra state "2020-12-25" -> "25-12-2020"
export const reverseString = str => {
    if (str !== undefined) {
        const splitString = str.split('-');
        const reverseArray = splitString.reverse();
        const joinedString = reverseArray.join('-');
        return joinedString;
    }
};
