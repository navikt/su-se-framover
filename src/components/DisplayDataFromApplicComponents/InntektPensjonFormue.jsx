import { Undertittel } from 'nav-frontend-typografi';
import { jaNeiSpørsmål } from './HelperFunctions';
import React from 'react';

const InntektPensjonFormue = ({ state }) => {
    return (
        <div style={headerSpacing}>
            <Undertittel style={elementSpacing}>Inntekt, pensjon, og formue</Undertittel>

            <div style={sectionGridLayout}>
                <span style={elementSpacing}>
                    Har søker fremsatt krav om annen norsk eller utenlandsk ytelse/pensjon som ikke er avgjort?
                </span>
                <span>{jaNeiSpørsmål(state.inntektPensjonFormue.framsattKravAnnenYtelse)}</span>
                <span>Hva slags ytelse/pensjon</span>
                <span>
                    {state.inntektPensjonFormue.framsattKravAnnenYtelse
                        ? state.inntektPensjonFormue.framsattKravAnnenYtelseBegrunnelse
                        : ''}
                </span>
            </div>

            <div style={sectionGridLayout}>
                <span style={elementSpacing}>Har du arbeidsinntekt/personinntekt?</span>
                <span>{jaNeiSpørsmål(state.inntektPensjonFormue.harInntekt)}</span>
                <span style={elementSpacing}>Brutto beløp per år:</span>
                <span>{state.inntektPensjonFormue.inntektBeløp}</span>
                <span style={elementSpacing}>Har du pensjon?</span>
                <span>{state.inntektPensjonFormue.harPensjon && søkerHarPensjon()}</span>
                <span>Sum inntekt:</span>
                <span>{state.inntektPensjonFormue.sumInntektOgPensjon}</span>
            </div>

            <div style={sectionGridLayout}>
                <span style={elementSpacing}>Har du formue/eiendom</span>
                <span>{jaNeiSpørsmål(state.inntektPensjonFormue.harFormueEiendom)}</span>
                <span style={elementSpacing}>Har du finansformue?</span>
                <span>{jaNeiSpørsmål(state.inntektPensjonFormue.harFinansFormue)}</span>
                <span style={elementSpacing}>
                    {state.inntektPensjonFormue.harFormueEiendom || state.inntektPensjonFormue.harFinansFormue ? (
                        <span>Totalbeløp formue:</span>
                    ) : (
                        ''
                    )}
                </span>
                <span>
                    {state.inntektPensjonFormue.harFormueEiendom || state.inntektPensjonFormue.harFinansFormue
                        ? state.inntektPensjonFormue.formueBeløp
                        : ''}
                </span>
                <span style={elementSpacing}>Har du annen formue/eiendom</span>
                <span>{jaNeiSpørsmål(state.inntektPensjonFormue.harAnnenFormue)}</span>
                <span>{state.inntektPensjonFormue.harAnnenFormue ? <span>-</span> : ''}</span>
                <span>{state.inntektPensjonFormue.harAnnenFormue ? søkerHarAnnenFormueEiendom() : ''}</span>
            </div>

            <div style={sectionGridLayout}>
                <span style={elementSpacing}>Har søker Depositumskonto?</span>
                <span>{jaNeiSpørsmål(state.inntektPensjonFormue.harDepositumskonto)}</span>
                <span>{state.inntektPensjonFormue.harDepositumskonto && <span>depositum-beløp:</span>}</span>
                <span>
                    {state.inntektPensjonFormue.harDepositumskonto && (
                        <span>{state.inntektPensjonFormue.depositumBeløp}</span>
                    )}
                </span>
            </div>

            <div style={sectionGridLayout}>
                <span>
                    Mottar du eller ektefellen/samboer, eller har du eller han/hun i løpet av de siste tre månedene
                    mottatt sosialstønad til livsopphold?
                </span>
                <span>{jaNeiSpørsmål(state.inntektPensjonFormue.harSosialStønad)}</span>
            </div>
        </div>
    );

    function søkerHarPensjon() {
        const array = state.inntektPensjonFormue.pensjonsOrdning;
        return (
            <ol>
                {array.map((pensjonsOrdningRow, index) => (
                    <li style={elementSpacing} key={index}>
                        {pensjonsOrdningRow.ordning}, {pensjonsOrdningRow.beløp}
                    </li>
                ))}
            </ol>
        );
    }

    function søkerHarAnnenFormueEiendom() {
        const array = state.inntektPensjonFormue.annenFormueEiendom;
        return (
            <ol>
                {array.map((annenFormueEiendomRow, index) => (
                    <li style={elementSpacing} key={index}>
                        {annenFormueEiendomRow.typeFormue},{annenFormueEiendomRow.skattetakst}
                    </li>
                ))}
            </ol>
        );
    }
};

const sectionGridLayout = {
    marginBottom: '1em',
    display: 'grid',
    gridTemplateColumns: '1fr 1fr'
};

const headerSpacing = {
    marginBottom: '2em'
};

const elementSpacing = {
    marginBottom: '1em'
};

export default InntektPensjonFormue;
