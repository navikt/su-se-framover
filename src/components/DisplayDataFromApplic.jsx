import { Systemtittel } from 'nav-frontend-typografi';
import React from 'react';
import Personopplysninger from './DisplayDataFromApplicComponents/Personopplysninger';
import Boforhold from './DisplayDataFromApplicComponents/Boforhold';
import Utenlandsopphold from './DisplayDataFromApplicComponents/Utenlandsopphold';
import Oppholdstillatelse from './DisplayDataFromApplicComponents/Oppholdstillatelse';
import InntektPensjonFormue from './DisplayDataFromApplicComponents/InntektPensjonFormue';
import ForNAV from './DisplayDataFromApplicComponents/ForNAV';

const DisplayDataFromApplic = ({ state }) => {
    console.log(state);
	if(Object.keys(state).length == 0) return null;
    return (
        <div>
            <Systemtittel style={{ marginBottom: '1em' }}>Oppsumerings side</Systemtittel>
            <Personopplysninger state={state} />
            <Boforhold state={state} />
            <Utenlandsopphold state={state} />
            <Oppholdstillatelse state={state} />
            <InntektPensjonFormue state={state} />
            <ForNAV state={state} />
        </div>
    );
};

export default DisplayDataFromApplic;
