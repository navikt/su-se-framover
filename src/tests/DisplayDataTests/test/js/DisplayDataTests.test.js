import React from 'react';
import { create } from 'react-test-renderer';
import DisplayDataFromApplic from '../../../../components/DisplayDataFromApplic.jsx';
import InntektPensjonFormue from '../../../../søknad/steg/InntektPensjonFormue';
import {jaNeiSpørsmål} from "../../../../HelperFunctions";

// prettier-ignore
const correctState = { personopplysninger: { fnr: "12345678910",  fornavn: "kake",  mellomnavn: "kjeks", etternavn: "mannen", telefonnummer: "12345678", gateadresse: "gaten", postnummer: "0050", poststed: "Oslo", bruksenhet: "50", bokommune: "Oslo", flyktning: true, borFastINorge: true, statsborgerskap: "NOR"}, boforhold: {delerBolig: true, borSammenMed: ["Ektefelle/Partner/Samboer", "Andre personer over 18 år"], delerBoligMed: [{fnr: "12345678910", navn: "voksen jensen"}, {fnr: "10987654321", navn: "voksen hansen"}]}, utenlandsopphold: {utenlandsopphold: true, antallRegistrerteDager: 11, registrertePerioder: [{utreisedato: "2020-03-10", innreisedato: "2020-03-10"}], planlagteUtenlandsopphold: true, antallPlanlagteDager: 10, planlagtePerioder: [{utreisedato: "2020-06-01", innreisedato: "2020-06-20"}]}, oppholdstillatelse: {harVarigOpphold: false, utløpsdato: "2020-03-10", søktOmForlengelse: true}, inntektPensjonFormue: {framsattKravAnnenYtelse: true, framsattKravAnnenYtelseBegrunnelse: "annen ytelse begrunnelse", harInntekt: true, inntektBeløp: 2500, harPensjon: true, pensjonsOrdning: [{ordning: "KLP", beløp: 2000}, {ordning: "SPK", beløp: 5000}], sumInntektOgPensjon: 7000, harFormueEiendom: true, harFinansFormue: true, formueBeløp: 1000, harAnnenFormue: true, annenFormue: [{typeFormue: "juveler", skattetakst: 2000}]}, forNav: {målform: "Bokmål", søkerMøttPersonlig: true, harFullmektigMøtt: false, erPassSjekket: true, merknader: "intet å bemerke"}}
// prettier-ignore
const stateWithError = { personopplysninger: { fnr: "12345678910",  fornavn: "kae",  mellomnavn: "kjeks", etternavn: "mannen", telefonnummer: "12345678", gateadresse: "gaten", postnummer: "0050", poststed: "Oslo", bruksenhet: "50", bokommune: "Oslo", flyktning: true, borFastINorge: true, statsborgerskap: "NOR"}, boforhold: {delerBolig: true, borSammenMed: ["Ektefelle/Partner/Samboer", "Andre personer over 18 år"], delerBoligMed: [{fnr: "12345678910", navn: "voksen jensen"}, {fnr: "10987654321", navn: "voksen hansen"}]}, utenlandsopphold: {utenlandsopphold: true, registrertePerioder: [{utreisedato: "2020-03-10", innreisedato: "2020-03-10"}], planlagteUtenlandsopphold: true, planlagtePerioder: [{utreisedato: "2020-06-01", innreisedato: "2020-06-20"}]}, oppholdstillatelse: {harVarigOpphold: false, utløpsdato: "2020-03-10", søktOmForlengelse: true}, inntektPensjonFormue: {framsattKravAnnenYtelse: true, framsattKravAnnenYtelseBegrunnelse: "annen ytelse begrunnelse", harInntekt: true, inntektBeløp: 2500, harPensjon: true, pensjonsOrdning: [{ordning: "KLP", beløp: 2000}, {ordning: "SPK", beløp: 5000}], sumInntektOgPensjon: 7000, harFormueEiendom: true, harFinansFormue: true, formueBeløp: 1000, harAnnenFormue: true, annenFormue: [{typeFormue: "juveler", skattetakst: 2000}]}, forNav: {målform: "Bokmål", søkerMøttPersonlig: true, harFullmektigMøtt: false, erPassSjekket: true, merknader: "intet å bemerke"}}

expect.extend({
    isNumber(received, number) {
        console.log('number', number);
        const bool = typeof number === 'number';
        console.log('bool', bool);
        if (bool) {
            return {
                message: () => 'The input given is a number',
                pass: true
            };
        } else {
            return {
                message: () => 'The input given is not a number',
                pass: false
            };
        }
    }
});

describe('Tests related to displaying data from a filled application', () => {
    test('Should match its snapshot', () => {
        const component = create(<DisplayDataFromApplic state={correctState} />);
        expect(component.toJSON()).toMatchSnapshot();
    });

    //Tall som kan begynne med null, vil fortsatt bli behandlet som strings
    test('JSON values that are numbers, should not be strings ', () => {
        const inntektPensjonFormue = create(<InntektPensjonFormue state={correctState} />);
        console.log(inntektPensjonFormue.toTree().props.state.inntektPensjonFormue);

        if (
            inntektPensjonFormue.toTree().props.state.inntektPensjonFormue.harPensjon ||
            inntektPensjonFormue.toTree().props.state.inntektPensjonFormue.harInntekt
        ) {
            expect().isNumber(inntektPensjonFormue.toTree().props.state.inntektPensjonFormue.sumInntektOgPensjon);
        }

        if (inntektPensjonFormue.toTree().props.state.inntektPensjonFormue.harPensjon) {
            inntektPensjonFormue
                .toTree()
                .props.state.inntektPensjonFormue.pensjonsOrdning.forEach(obj => expect().isNumber(obj.beløp));
        }

        if (inntektPensjonFormue.toTree().props.state.inntektPensjonFormue.harInntekt) {
            expect().isNumber(inntektPensjonFormue.toTree().props.state.inntektPensjonFormue.inntektBeløp);
        }

        if (
            inntektPensjonFormue.toTree().props.state.inntektPensjonFormue.harFormueEiendom ||
            inntektPensjonFormue.toTree().props.state.inntektPensjonFormue.harFinansFormue
        ) {
            expect().isNumber(inntektPensjonFormue.toTree().props.state.inntektPensjonFormue.formueBeløp);
        }

        if (inntektPensjonFormue.toTree().props.state.inntektPensjonFormue.harAnnenFormue) {
            inntektPensjonFormue
                .toTree()
                .props.state.inntektPensjonFormue.annenFormue.forEach(obj => expect().isNumber(obj.skattetakst));
        }

        if (inntektPensjonFormue.toTree().props.state.inntektPensjonFormue.harDepositumskonto) {
            expect().isNumber(inntektPensjonFormue.toTree().props.state.inntektPensjonFormue.depositumBeløp);
        }
    });

    test('Tests that the incoming state has the correct properties', () => {
        const component = create(<DisplayDataFromApplic state={correctState} />);
        expect(component.toTree().props.state).toHaveProperty(
            'forNav' &&
                'boforhold' &&
                'inntektPensjonFormue' &&
                'personopplysninger' &&
                'utenlandsopphold' &&
                'oppholdstillatelse'
        );
    });

    test('Tests that the component has exactly 7 children', () => {
        const component = create(<DisplayDataFromApplic state={correctState} />);
        //the component should have 7 children.
        expect(component.toJSON().children.length).toBe(7);
    });

    test('Tests that two DisplayDataFromApplic components with a difference in målform are not equal', () => {
        const component = create(<DisplayDataFromApplic state={correctState} />);
        const component2 = create(<DisplayDataFromApplic state={stateWithError} />);

        expect(component.toJSON()).not.toEqual(component2.toJSON());
    });

    test('jaNeiSpørsmål that changes Boolean to ja/nei, should return the correct results for the given Boolean', () => {
        let state = {key: true};
        let result = jaNeiSpørsmål(state.key);
        expect(result).toBe("Ja");

        state.key = false;
        let nei = jaNeiSpørsmål(state.key);
        expect(nei).toBe("Nei");

        state.key = {};
        result = jaNeiSpørsmål(state.key);
        expect(result).toBeFalsy;
    });
});
