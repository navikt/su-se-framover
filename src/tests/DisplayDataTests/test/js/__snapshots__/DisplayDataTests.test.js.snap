// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests related to displaying data from a filled application Should match its snapshot 1`] = `
<div>
  <h2
    className="typo-systemtittel"
    style={
      Object {
        "marginBottom": "1em",
      }
    }
  >
    Oppsumerings side
  </h2>
  <div
    style={
      Object {
        "marginBottom": "2em",
      }
    }
  >
    <h2
      className="typo-undertittel"
      style={
        Object {
          "marginBottom": "1em",
        }
      }
    >
      Personopplysninger
    </h2>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Fødselsnummer:
      </span>
      <span>
        12345678910
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Fornavn:
      </span>
      <span>
        kake
      </span>
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Mellomnavn:
      </span>
      <span>
        kjeks
      </span>
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Etternavn:
      </span>
      <span>
        mannen
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span>
        Telefonnummer:
      </span>
      <span>
        12345678
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Gateadresse:
      </span>
      <span>
        gaten
      </span>
      <span>
        Bruksenhet:
      </span>
      <span>
        50
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Postnummer:
      </span>
      <span>
        0050
      </span>
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Poststed:
      </span>
      <span>
        Oslo
      </span>
      <span>
        Bokommune:
      </span>
      <span>
        Oslo
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span>
        Søkers statsborkerskap:
      </span>
      <span>
        NOR
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Er søker registrert som flyktning?
      </span>
      <span>
        Ja
      </span>
      <span>
        Bor søker fast i Norge?
      </span>
      <span>
        Ja
      </span>
    </div>
  </div>
  <div
    style={
      Object {
        "marginBottom": "2em",
      }
    }
  >
    <h2
      className="typo-undertittel"
      style={
        Object {
          "marginBottom": "1em",
        }
      }
    >
      Boforhold
    </h2>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Deler søker bolig med en annen voksen?
      </span>
      <span>
        Ja
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span>
        Søker deler bolig med:
      </span>
      <span>
        Ektefelle/Samboer/Partner, Andre personer over 18
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span>
        Opplysninger:
      </span>
      <span>
        <ol>
          <li
            style={
              Object {
                "marginBottom": "1em",
              }
            }
          >
            12345678910
             
            voksen jensen
          </li>
          <li
            style={
              Object {
                "marginBottom": "1em",
              }
            }
          >
            10987654321
             
            voksen hansen
          </li>
        </ol>
      </span>
    </div>
  </div>
  <div
    style={
      Object {
        "marginBottom": "2em",
      }
    }
  >
    <h2
      className="typo-undertittel"
      style={
        Object {
          "marginBottom": "1em",
        }
      }
    >
      Utenlandsopphold
    </h2>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span>
        Registrert utenlandsopphold:
      </span>
      <span>
        Ja
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <label>
        Utenlandsopphold:
      </label>
      <label>
         
        <ol>
          <li
            style={
              Object {
                "marginBottom": "1em",
              }
            }
          >
            10-03-2020
             -  
            10-03-2020
          </li>
        </ol>
      </label>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <label
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Antall registrerte dager:
      </label>
      <label>
         
        11
      </label>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span>
        Planlagt utenlandsopphold:
      </span>
      <span>
        
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    />
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    />
  </div>
  <div
    style={
      Object {
        "marginBottom": "2em",
      }
    }
  >
    <h2
      className="typo-undertittel"
      style={
        Object {
          "marginBottom": "1em",
        }
      }
    >
      Opplysninger om oppholdstillatelse
    </h2>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span>
        Har søker varig oppholdstillatelse:
      </span>
      <span>
        Nei
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span>
        Utløpsdato: 
      </span>
      <span>
        10-03-2020
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span>
        Har søker søkt om forlengelse?
      </span>
      <span>
        Ja
      </span>
    </div>
  </div>
  <div
    style={
      Object {
        "marginBottom": "2em",
      }
    }
  >
    <h2
      className="typo-undertittel"
      style={
        Object {
          "marginBottom": "1em",
        }
      }
    >
      Inntekt, pensjon, og formue
    </h2>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Har søker fremsatt krav om annen norsk eller utenlandsk ytelse/pensjon som ikke er avgjort?
      </span>
      <span>
        Ja
      </span>
      <span>
        Hva slags ytelse/pensjon
      </span>
      <span>
        annen ytelse begrunnelse
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Har du arbeidsinntekt/personinntekt?
      </span>
      <span>
        Ja
      </span>
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Brutto beløp per år:
      </span>
      <span>
        2500
      </span>
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Har du pensjon?
      </span>
      <span>
        Ja
      </span>
      <span />
      <span>
        <ol>
          <li
            style={
              Object {
                "marginBottom": "1em",
              }
            }
          >
            KLP
            , 
            2000
          </li>
          <li
            style={
              Object {
                "marginBottom": "1em",
              }
            }
          >
            SPK
            , 
            5000
          </li>
        </ol>
      </span>
      <span>
        Sum inntekt og pensjon:
      </span>
      <span>
        7000
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Har du formue/eiendom
      </span>
      <span>
        Ja
      </span>
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Har du finansformue?
      </span>
      <span>
        Ja
      </span>
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        <span>
          Totalbeløp formue:
        </span>
      </span>
      <span>
        1000
      </span>
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Har du annen formue/eiendom
      </span>
      <span>
        Ja
      </span>
      <span>
        <span>
          -
        </span>
      </span>
      <span>
        <ol>
          <li
            style={
              Object {
                "marginBottom": "1em",
              }
            }
          >
            juveler
            ,
            2000
          </li>
        </ol>
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Har søker Depositumskonto?
      </span>
      <span>
        
      </span>
      <span />
      <span />
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span>
        Mottar du eller ektefellen/samboer, eller har du eller han/hun i løpet av de siste tre månedene mottatt sosialstønad til livsopphold?
      </span>
      <span>
        
      </span>
    </div>
  </div>
  <div
    style={
      Object {
        "marginBottom": "2em",
      }
    }
  >
    <h2
      className="typo-undertittel"
      style={
        Object {
          "marginBottom": "1em",
        }
      }
    >
      For NAV
    </h2>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span>
        Hvilket målform ønsker du svaret i?
      </span>
      <span>
        Bokmål
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Har søker møtt personlig?
      </span>
      <span>
        Ja
      </span>
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Har fullmektig møtt?
      </span>
      <span>
        Nei
      </span>
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        Er originalt(e) pass sjekket for stempel?
      </span>
      <span>
        Ja
      </span>
    </div>
    <div
      style={
        Object {
          "display": "grid",
          "gridTemplateColumns": "1fr 1fr",
          "marginBottom": "1em",
        }
      }
    >
      <span
        style={
          Object {
            "marginBottom": "1em",
          }
        }
      >
        <span>
          Merknader:
        </span>
      </span>
      <span>
        <span>
          intet å bemerke
        </span>
      </span>
    </div>
  </div>
</div>
`;
